docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+
+services:
+  ntt:
+    depends_on:
+        - postgres
+    build:
+      context: .
+    restart: on-failure
+    environment:
+      - NEW_BUILD=true
```

Comment:
this is currently turned on by default but should be set to false after the first run.

Reply:
with docker-compose Im not sure it makes sense to use build-arg.